### PR TITLE
Php questions importance marks

### PR DESCRIPTION
**Pull request by a-kharunzha**
Based and dependent on 
https://github.com/SharpMindsUA/CompetencyMatrix/pull/5
https://github.com/SharpMindsUA/CompetencyMatrix/pull/6
https://github.com/SharpMindsUA/CompetencyMatrix/pull/7
https://github.com/SharpMindsUA/CompetencyMatrix/pull/8

### CODE REVIEW

File: PHP/Language.md [hunk 1]
```diff
@@ -7,34 +7,34 @@ Write data to the script’s output: echo vs print();
...
 Variables (naming, case-sensitivity). Variable variables.
 
-Constants. Possible ways to define. Possible values that they contain. Magic constants.
+Constants. Possible ways to define. Possible values that they contain. Magic constants. (*) 
 
 ### Operators
 Operator precedence.
 
-Arithmetic operators ($a**2 etc)
+Arithmetic operators ($a*2 etc)
```
Comment (BohdanKyryliuk) [inline PHP/Language.md]: Do you really need replacing **Exponentiation** operator by **Multiplication**?

Comment (a-kharunzha) [reply PHP/Language.md]: Ha-ha, no :D. I just thought that it was a typo for a moment. I have never used it before. Will revert then, thanks!

File: PHP/Git.md [hunk 2]
```diff
@@ -3,17 +3,17 @@ Version control system concept
...
 
-Branches and commits
+Branches and commits (*)
 
 Moving up and down through you repo with `git checkout`
 
-Merging
+Merging (*)
 
 Undoing changes with `git reset` and `git revert`
```
Comment (OlegGin) [inline PHP/Git.md]: I would ask about this

File: PHP/Language.md [hunk 3]
```diff
@@ -7,34 +7,37 @@ Write data to the script’s output: echo vs print();
 
 Script termination. 
 
-Data types: 4 scalar types, 4 compound types, 2 special types. Pseudo-types. 
+Data types: 4 scalar types, 4 compound types, 2 special types. Pseudo-types. (*) 
 
-Type Casting (with casting operators and functions). Detecting Types. 
+Type Casting (with casting operators and functions). Detecting Types. (*)  
 
 Variables (naming, case-sensitivity). Variable variables.
```
Comment (OlegGin) [inline PHP/Language.md]: I would ask about case-sensitivity.
May be split this line into two questions?

File: PHP/Language.md [hunk 4]
```diff
@@ -45,34 +48,41 @@ if, while, for, foreach, switch and their alternative syntax.
 
 continue and break statements
```
Comment (OlegGin) [inline PHP/Language.md]: I would ask about this

File: PHP/Language.md [hunk 5]
```diff
@@ -97,17 +111,23 @@ $_GET, $_POST, $_REQUEST
```
Comment (OlegGin) [inline PHP/Language.md:113]: I would ask about difference $_GET, $_POST, $_REQUEST

File: PHP/Language.md [hunk 6]
```diff
@@ -97,17 +111,23 @@ $_GET, $_POST, $_REQUEST
 
 $_FILES
 
-Cookies and sessions in php
+Cookies and sessions in php (*)
 
 ### OOP Fundamentals
 Class vs object
```
Comment (OlegGin) [inline PHP/Language.md]: I would ask about this.
This is very important question. Some developers confused.-)

File: PHP/Language.md (hunk 5 above)
Comment (a-kharunzha) [reply PHP/Language.md:113]: Yeah, It's basics. Actually, did not include them in the first edition, as nowadays all people used to work with Request objects. But will add now.

**Pull request merged**